### DISCUSSION TITLE
Maximum update depth exceeded from getItemKey

### DISCUSSION THREAD

**Discussion by issimin** [TanStack/virtual#789] [Q&A]
const getItemKey = useCallback(
            (index: number) => {
                return (item && itemKey?.(item)) ?? index;
            },
            [itemKey, items] 
        );
        

In the performance pressure test scenario, this 'Maximum update ' error is reported with 'items', but not without 'items'. How can I solve this problem

**Comment by piecyk** [comment]
@issimin check if itemKey and items are stale.
